src/plugins/image_embeddable/kibana.jsonc

```diff
@@ -7,7 +7,7 @@
     "id": "imageEmbeddable",
     "server": false,
     "browser": true,
-    "requiredPlugins": ["embeddable", "files", "uiActions", "kibanaReact"],
+    "requiredPlugins": ["embeddable", "files", "uiActions", "kibanaReact", "dashboard"],
```

Comment:
TBH, I'm surprised that we are not running into circular dependencies 😮

Reply:
I thought about this, wouldn't the ui-action plugin be a more appropriate place to place this constant? I see that there's an existing pattern of creating triggers and exposing them through the `uiAction` plugin.